src/OpenTelemetry/CHANGELOG.md

```diff
@@ -56,6 +56,11 @@ Released 2021-Mar-19
 * Added `SetResourceBuilder` support to `OpenTelemetryLoggerOptions`.
   ([#1913](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1913))
 
+* Use `AssemblyFileVersionAttribute` instead of `FileVersionInfo.GetVersionInfo`
```

Comment:
Could you move this to the Unreleased section? (We released 1.1.0-beta1 last week, and this change will be part of the next release only)

Reply:
Ah, didn't see that section. Moved.